great_tables/_utils_render_html.py

```diff
@@ -39,22 +39,20 @@ def create_heading_component_h(data: GTData) -> StringBuilder:
     )
 
     if has_subtitle:
-        heading = f"""
-  <tr>
-    <th colspan="{n_cols_total}" class="gt_heading gt_title gt_font_normal">{title}</th>
+        heading = f"""<tr class="gt_heading">
```

Comment:
Can you change this to match the previous formatting? (so there is a line break at the beginning of this string?)